cds_ils/ldap/serializers.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015-2018 CERN.
+#
+# cds-migrator-kit is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-ILS ldap serializers."""
+import sys
+
+from invenio_accounts.models import User
+from invenio_oauthclient.models import UserIdentity
+from invenio_userprofiles import UserProfile
+
+
+class LDAPUser:
+    """LDAPUser serializer class."""
+
+    def __init__(self, ldap_user_data, log_func=lambda x: x):
+        """Constructor."""
+        self.is_valid_user = True
+        self.is_duplicate = False
+        self.employee_id = ldap_user_data["employeeID"][0].decode("utf8")
+        self.data = {}
```

Comment:
Maybe I am wrong, the diff is large, but it looks to me that `data` is required not to be empty, when used.
Isn't it better to `raise` if `validate_required` return `False`?

Reply:
I was considering it while implementing, I will fix it since you think the same :)